vertx-pg-client/src/test/java/io/vertx/pgclient/PgPoolTest.java

```diff
@@ -422,6 +423,49 @@ public void testPoolIdleTimeout(TestContext ctx) {
       .onComplete(ctx.asyncAssertSuccess());
   }
 
+  @Test
+  public void testPoolMaxLifetime(TestContext ctx) {
+    ProxyServer proxy = ProxyServer.create(vertx, options.getPort(), options.getHost());
+    AtomicReference<ProxyServer.Connection> proxyConn = new AtomicReference<>();
+    int pooleCleanerPeriod = 100;
```

Comment:
How about taking the opportunity to rename this to `poolCleanerPeriod` in this test and others?

Reply:
done